pkg/api/componentreadiness/component_report.go

```diff
@@ -395,18 +422,176 @@ func (c *componentReportGenerator) getTestStatusFromBigQuery(ctx context.Context
 		case <-ctx.Done():
 			return
 		default:
-			sampleStatus, sampleErrs = c.getSampleQueryStatus(ctx, allJobVariants)
+			includeVariants, skipQuery := copyIncludeVariantsAndRemoveOverrides(c.variantJunitTableOverrides, -1, c.IncludeVariants)
+			if skipQuery {
+				fLog.Infof("skipping default status query as all values for a variant were overridden")
+				return
+			}
+			fLog.Infof("running default status query with includeVariants: %+v", includeVariants)
+			status, errs := c.getSampleQueryStatus(ctx, allJobVariants, includeVariants, c.SampleRelease.Start, c.SampleRelease.End, defaultJunitTable)
+			fLog.Infof("received %d test statuses and %d errors from default query", len(status), len(errs))
+			statusCh <- status
+			for _, err := range errs {
+				statusErrCh <- err
+			}
 		}
 
 	}()
 
-	wg.Wait()
+	// fork additional sample queries for the overrides
```

Comment:
I'm pretty sure this breaks if you tried to do more than one override due to the includeVariant logic being an AND.